src/borg/archiver.py

```diff
@@ -2358,64 +2358,64 @@ def do_break_lock(self, args, repository):
             EOF
             $ borg create --exclude-from exclude.txt backup /
 
```

Comment:
The below regression was introduced in 5d4dda75d36d4bdaff9fc4a38a234383aef4b733 - nothing to backport

Reply:
do you mean the bigger block with bad indentation? i don't see it there.

Reply:
Sorry. My comment got stuck because of the github review feature and I did not notice.

Reply:
Thanks for the hint!

Reply:
removed backport label, nothing to do here...